sdk/include/opentelemetry/sdk/trace/batch_span_processor.h

```diff
@@ -37,6 +37,12 @@ struct BatchSpanProcessorOptions
    * equal to max_queue_size.
    */
   size_t max_export_batch_size = 512;
+
+  /**
+   * Determines whether the export happens asynchronously.
+   * Default implementation is synchronous.
+   */
+  bool is_export_async = false;
```

Comment:
Should the async interface also added for simple_span_processor?

Reply:
Agreed on this, will make the change.